src/ElmFormat.hs

```diff
@@ -223,6 +229,7 @@ main'' elmFormatVersion_ experimental_ args =
                 do
                     let autoYes = Flags._yes config
                     resolvedInputFiles <- Execute.run (Execute.forHuman autoYes) $ resolveFiles (Flags._input config)
+                    detectedElmVersion <- Execute.run (Execute.forHuman autoYes) $ ElmVersion.fromFile "."
```

Comment:
Can this IO action fail?  If so, we should maybe postpone running this until we know that `detectedElmVersion` will be used.

Reply:
This is still needed in last version, but things have changed a lot, so let's discuss it again based on the new version.